demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -95,3 +97,22 @@ def is_html_doc(self) -> bool:
             return True
         # use some heuristics to try to figure out if this is html
         return txt.startswith('<p>') or ('<thead>' in txt and '<tbody>' in txt)
+
+    def is_image_path_valid(self) -> bool:
+        with open(self.file_path) as f:
+            readme_content = f.read()
+            invalid_paths = re.findall(
+                r'(\!\[.*?\]|src\=)((\(|\")https://github.com/demisto/content/(?!raw).*?(\)|\"))', readme_content,
+                re.IGNORECASE)
+            if invalid_paths:
+                for path in invalid_paths:
+                    # Grabbing only url and removing " " or ( ) url wrapper
+                    path = path[1][1:-1]
+                    alternative_path = path.replace('blob', 'raw')
+                    print_error(f'============ Detected following image url: ============\n{path} \n'
```

Comment:
why isn't that print part of the error object?

Reply:
It seemed like a lot of data at the end of the validation process. So I figured it would be more readable this way.
Should I put it in the error object instead?

Reply:
add it to the error object, in the error summary only the error code is presented so it will not spam that section